editor/sanitize.js

```diff
@@ -154,17 +153,16 @@ svgedit.sanitize.sanitizeSvg = function(node) {
       var attrName = attr.nodeName;
       var attrLocalName = attr.localName;
       var attrNsURI = attr.namespaceURI;
+
+      // Ignore attributes from custom namespaces
+      var splitNS = attrName.match(/([^:]+):/);
+      if(splitNS && splitNS[1].toUpperCase() in svgedit.ignoredNS) continue;
```

Comment:
Perhaps this is this what's going on...
The svgedit.NS.* properties are upper case because they are constants. The rest of the svgedit codebase assumes namespaces prefixes are always lower case (incorrectly?), so getReverseNS() calls toLower() on each property name. Given all that, line 159 above converts the namespace prefix on each attribute to the upper-case object property name.

Given all that, this code works the same as the rest of the codebase.